src/main/java/org/opentripplanner/standalone/GraphBuilderParameters.java

```diff
@@ -198,6 +198,18 @@
      */
     public boolean writeCachedElevations;
 
+    /**
+     * When set to true (it is false by default), the elevation module will include the Ellipsoid to Geiod difference in
+     * the calculations of every point along every StreetWithElevationEdge in the graph.
+     */
+    public boolean includeEllipsoidToGeoidDifference;
+
+    /**
+     * The number of significant digits to use when caching Ellipsoid to Geiod difference values. The defualt is 0 and
+     * is capped at 2.
+     */
```

Comment:
This is the number of significant digits to use for x,y values, correct? Maybe just add that detail.